config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-capi-openstack/capi-openstack-controller:dev
+      - image: gcr.io/k8s-staging-capi-openstack/capi-openstack-controller:latest
```

Comment:
previously we will update this file (so that git will ask  you whether to commit after some test)
after this change, this file will not be modified when doing generate.sh, right?

Reply:
I think generate.sh didn't ever update this file. But some docker build/push targets in the Makefile did and I removed them :)

Reply:
I remember wrong thing, you are right, make command will do the change :) and thanks for confirm it's removed~